storage_broker/src/metrics.rs

```diff
@@ -0,0 +1,25 @@
+//! Broker metrics.
+
+use metrics::{register_int_gauge, IntGauge};
+use once_cell::sync::Lazy;
+
+pub static NUM_PUBS: Lazy<IntGauge> = Lazy::new(|| {
+    register_int_gauge!("broker_active_publishers", "Number of publications")
```

Comment:
Could we prefix these with `storage_broker_` instead of just plain ` broker`? We have `pageserver` metrics with `pageserver_` prefix, `safekeeper` metrics with `safekeeper_` prefix and `proxy` metrics with `proxy_` prefix. I won't insist, though, as it's not that big of a deal.

Reply:
Yes, renamed.